iBudget/ibudget/views.py

```diff
@@ -0,0 +1,43 @@
+"""
+Views module for handling AWS S3
+"""
+from django.http import HttpResponse
+from django.views import View
+from utils.aws_helper import upload, get_image_url, del_photo, check_if_in_bucket
+from utils.response_helper import (RESPONSE_400_INVALID_DATA)
+
+
+class AwsCall(View):
+    """ View for handling CRUD methods for images in the AmazonS3 bucket
+    """
+
+    def post(self, request):
+        """The name property of the file which is passed is 'icon', so in HTML form it must be set:
+        <input type='file' name = 'icon'>"""
+        file = request.FILES['icon']
+        if check_if_in_bucket(file):
+            upload(file)
+            file = str(file)
+            url = get_image_url(file)
+            return HttpResponse(url)
+        return RESPONSE_400_INVALID_DATA
+
+    @staticmethod
+    def put(request):
+        """The method must accept the name of the file (string) of the image which is already used
+        as a profile photo, and the new image which is passed in a form
+        """
+        old_file = request.body.decode()
+        new_file = request.FILES['icon']
+        del_photo(old_file)
```

Comment:
It looks like you do not handle status of `del_photo(old_file)` and `upload(new_file)`. What if one of them or both are not successful?

Reply:
Please specify what exactly could be improved there. Should I check the status code of the response from the server and if it's not 200, 201 or 204 the handler must return the response with the message like "the deletion/change was not successful"?